scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -370,38 +392,43 @@ sce_list_formatted <- purrr::map2(
   # Name of the function to run
   format_sce
 )
+```
 
+```{r print_sce_list_formatted, live = TRUE}
+# Print resulting list
 sce_list_formatted
 ```
 
 (Psst, like `purrr` and want to dive deeper? Check out [the `purrr::imap()` function](https://purrr.tidyverse.org/reference/imap.html)!)
 
 
-At long last, we are ready to merge the SCEs!
-We'll use the R function `cbind()` for this.
+At long last, we are ready to merge the SCEs, which we'll do using the R function `cbind()`.
 The `cbind()` function is often used to combine data frames or matrices by column, i.e. "stack" them next to each other.
 The same principle applies here, but when run on SCE objects, `cbind()` will create a new SCE object by combining `counts` and `logcounts` matrices by column.
 Following that structure, other SCE slots (`colData`, `rowData`, reduced dimensions, and other metadata) are combined appropriately.
 
 Since we need to apply `cbind()` to a _list_ of objects, we need to use some slightly-gnarly syntax: We'll use the function `do.call()`, which allows the `cbind()` input to be a list of objects to combine.
 
-```{r merge sces}
+```{r merges_sces, live = TRUE}
 # Merge SCE objects 
 merged_sce <- do.call(cbind, sce_list_formatted)
+```
+
 
-# Let's have a look!
+```{r print_merged_sce, live = TRUE}
+# Print the merged_sce object
```

Comment:
Same here... no need to split these chunks, I don't think, as only one has output. You might want to do it live, but I don't think it needs to be done in the published notebook.